docs/CONTRIBUTING.md

```diff
@@ -40,7 +40,7 @@ available in your local environment:
 
 If you want to run integration tests you will also need:
 
-* A serial port with an Arduino device attached
+* A serial port with an Arduino board attached
 * A working [Python][3] environment, version 3.5 or later
```

Comment:
```suggestion
* A working [Python][3] environment, version 3.8 or later
```

As we have here:
https://github.com/arduino/arduino-cli/blob/bf4042e67646405955fd05880b04ff12e7b37b55/.github/workflows/test.yaml#L71